### ISSUE TITLE
no answer from AI

### ISSUE THREAD

**Issue by thesayer3** [Rubiksman78/MonikA.I#51]
![Screenshot 2023-03-28 155548](https://user-images.githubusercontent.com/129194879/228273085-d4259342-8631-4648-92c7-a495e6bd60ef.png)

i type my question, and dont receive an answer. so the game isnt responding and i have to close it to talk to monika again.
with debug mode, nothing happens

**Comment by Rubiksman78** [comment]
Try running the `update.bat` and try again, be sure to delete `config.json` if it wasn't deleted by the updater.

**Comment by thesayer3** [comment]
> Try running the `update.bat` and try again, be sure to delete `config.json` if it wasn't deleted by the updater.

after i did that, i launched game and typed my question but she said "there is a bug somewhere and then programs wont respond, i restored the proccess and the game showed an error:

```python
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Submods/AI_submod/monikai_chat.rpy", line 169, in script call
    call monikai_get_actions
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/event-handler.rpy", line 3119, in script call
  File "game/Submods/AI_submod/monikai_emotion.rpy", line 79, in script
    $ received_emotion = receiveMessage()
  File "game/Submods/AI_submod/monikai_emotion.rpy", line 79, in <module>
    $ received_emotion = receiveMessage()
  File "game/Submods/AI_submod/monikai_chat.rpy", line 10, in receiveMessage
    msg = clientSocket.recv(BUFSIZ).decode("utf8")
error: [Errno 10053] An established connection was aborted by the software in your host machine

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Submods/AI_submod/monikai_chat.rpy", line 169, in script call
    call monikai_get_actions
  File "game/script-ch30.rpy", line 2141, in script call
  File "game/event-handler.rpy", line 3119, in script call
  File "game/Submods/AI_submod/monikai_emotion.rpy", line 79, in script
    $ received_emotion = receiveMessage()
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/Submods/AI_submod/monikai_emotion.rpy", line 79, in <module>
    $ received_emotion = receiveMessage()
  File "game/Submods/AI_submod/monikai_chat.rpy", line 10, in receiveMessage
    msg = clientSocket.recv(BUFSIZ).decode("utf8")
error: [Errno 10053] An established connection was aborted by the software in your host machine

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.14
``` 

and terminal said that:

```ren'py
Exception in thread Thread-3:
Traceback (most recent call last):
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\libs\pythonlib\lib\threading.py", line 932, in _bootstrap_inner
    self.run()
  File "C:\Program Files (x86)\Steam\steamapps\common\Doki Doki Literature Club\libs\pythonlib\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "main.py", line 368, in listenToClient
    rest_msg = received_msg[1]
IndexError: list index out of range
``` 

when i was updating, terminal said this and instantly closed:

```
'git' is not recognized as an internal or external command, operable program or batch file.
'git' is not recognized as an internal or external command, operable program or batch file. WARNING: Requirement  '[simpleaudio-1.0.4-cp38-cp38-win_amd64.whl](http://simpleaudio-1.0.4-cp38-cp38-win_amd64.whl/) looks like a filename, but the file does not exist
Processing c:\program files (x86)\steam\steamapps\common\doki doki literature club\[simpleaudio-1.0.4-cp38-cp38-win_amd64.whl](http://simpleaudio-1.0.4-cp38-cp38-win_amd64.whl/) ERROR: Could not install packages due to an OSError: [Errno 2] No such file or directory: 'C:\\Program Files (x86)\\Steam\\steamapps\\common\\Doki Doki Literature Club\\[simpleaudio-1.0.4-cp38-cp38-win_amd64.whl](http://simpleaudio-1.0.4-cp38-cp38-win_amd64.whl/)
``` 


<sub>yes i run it on steam</sub>

**Comment by Rubiksman78** [comment]
Did you follow the installation steps from the wiki page for the user version since 1.6.1? You are missing a lot of files, git and others so it seems you haven't even done the installation correctly.

**Comment by thesayer3** [comment]
> Did you follow the installation steps from the wiki page for the user version since 1.6.1? You are missing a lot of files, git and others so it seems you haven't even done the installation correctly.

I downloaded git (but i dont know what this is and how to use it), ive installed both files from "releases", ive clicked and downloaded all in setup_gpu.bat . but i havent downloaded any TTS files.

**Comment by Rubiksman78** [comment]
Check your installation of git or do it again, it is not installed correctly

**Comment by popdanceytb** [comment]
same here i installed playwright 3 times but it keeps crashing and when i use the cai 1 or 2 she says that a bug is somewhere.

**Comment by Rubiksman78** [comment]
> same here i installed playwright 3 times but it keeps crashing and when i use the cai 1 or 2 she says that a bug is somewhere.

If it's crashing, please report correctly the error in the terminal, if it seems to be unrelated please submit another issue.
